Drv/TcpClient/test/ut/TcpClientTester.cpp

```diff
@@ -16,6 +16,7 @@
 #include <Drv/Ip/test/ut/SocketTestHelper.hpp>
 
 Os::Log logger;
+const U64 SOCKET_RETRY_INTERVAL_MS = (SOCKET_RETRY_INTERVAL.getSeconds() * 1000) + (SOCKET_RETRY_INTERVAL.getUSeconds()/1000);
```

Comment:
The arithmetic on the RHS may be 32-bit arithmetic. Given the U64 on the LHS, it looks like this isn't intended. I think a cast to U64 on the right is required, maybe before multiplying by 1000.

Reply:
aggreed!